imagine/pkg/main.go

```diff
@@ -102,7 +106,8 @@ func (conf *Config) Run() error {
 	return nil
 }
 
-func (conf *Config) readSpecs() error {
```

Comment:
The idea was that external programs can use `NewSpecsFile()` to specify its own specs.toml file and `ReadSpecs()` to populate the fields in a `Config` struct from that. Otherwise, the fields that `ReadSpecs()` populates are unexported and cannot be set without additional exports.